src/lib.rs

```diff
@@ -1466,9 +1548,34 @@ impl Rtc {
             time
         };
 
+        self.last_timeout_reason = reason;
+
         Ok(Output::Timeout(next))
     }
 
+    /// The reason for the last [`Output::Timeout`]
+    ///
+    /// This is updated when calling [`Rtc::poll_output()`] and the next output
+    /// is a timeout.
+    ///
+    /// ```
+    /// # use str0m::{Rtc, Input, Output, Reason};
+    /// // A vec holding the managed rtc instances. One instance per remote peer.
```

Comment:
A vec comment can probably be removed here

Reply:
thanks!